saas/urls/provider/metrics.py

```diff
@@ -42,6 +42,8 @@
         name='saas_subscriber_pipeline_download'),
     url(r'^plans/((?P<from_date>\d\d\d\d-\d\d)/)?',
         PlansMetricsView.as_view(), name='saas_metrics_plans'),
+    url(r'^coupons/_download/?$', CouponMetricsDownloadView.as_view(),
```

Comment:
No, sorry. The only reason I did that was to add differentiation against the following URL pattern:

```
url(r'^coupons/((?P<coupon>%s)/)?' % ACCT_REGEX,
        CouponMetricsView.as_view(), name='saas_metrics_coupons'),
```

This means that `_coupon` cannot be used as the code for a coupon, otherwise we'd have a URL mapping conflict.

Come to think of it, my most recent commit does the opposite of the above, here:
https://github.com/marmida/djaodjin-saas/commit/d63729485262c92aa6e1eb872f1e1754073dc03a

We should probably pick a unified approach to dealing with this, so that these URL schemes are consistent.